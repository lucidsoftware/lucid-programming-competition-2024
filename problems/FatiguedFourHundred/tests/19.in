version https://git-lfs.github.com/spec/v1
oid sha256:a92a97a5ac5325197930ed3b775e755102892f7d8be0fd476786fe8236288f40
size 482410
