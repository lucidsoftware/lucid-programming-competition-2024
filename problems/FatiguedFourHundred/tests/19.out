version https://git-lfs.github.com/spec/v1
oid sha256:b995ada5a59f74e309ee06408ec5616a2c845e8420940eececb88807324d9ead
size 356516
